pages/07.help/help.fr.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Aidez-moi
+template: docs
+taxonomy:
+    category: docs
+routes:
+  default: '/help-me'
+---
+
+Bien que les contributeurs de YunoHost essaient continuellement de l'améliorer, YunoHost ou sa documentation ne sont exempts de bugs ou d'erreurs. Pour nous aider à vous aider, vous devez cependant suivre ces instructions.
+
+Le support n'est prodigué que sur [le forum](https://forum.yunohost.org?target=_blank) ou sur [nos salons de discussion](/chat_rooms?target=_blank). 
+Nous sommes présents sur Mastodon, mais nous ne prodiguons pas d'aide sur cette plateforme.
+
+Pour vous guider, sur le forum, vous devez remplir le modèle de demande de support.
```

Comment:
```suggestion
Pour que nous puissions être en mesure de vous aider sur le forum, vous devez impérativement remplir le modèle de demande de support.
```

(I don't know if that was the intent of the original message ? If yes, to be propagated to english as well)